.github/workflows/jest-testing.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Jest testing suite
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Could you also add the `pull_request` handler?

Reply:
Of course. No filter needed, any pull request will do?

Reply:
> Of course. No filter needed, any pull request will do?

yes